tests/helix/send-to-helix-buildonhelixtests.targets

```diff
@@ -27,6 +27,10 @@
     <ItemGroup>
       <_TestRunCommandArguments Remove="@(_TestRunCommandArguments)" />
 
+      <_TestBlameArguments Remove="@(_TestBlameArguments)" />
```

Comment:
What's already in this item that need to be cleared?

Reply:
https://github.com/dotnet/aspire/blob/79cdd0688ada0a5d0b0c4e3b91cc7f06353648ff/tests/helix/send-to-helix-inner.proj#L66-L70

Reply:
Why don't we want the defaults? Maybe adding a comment here could help.

Reply:
I had this on the commit message, which is not useful enough as it would get lost in the squashed commit: `Remove the blame-hang-timeout for playground tests, as it is pretty much one main test running`.

I'll add a comment with that expanded to:
`Remove the blame-hang-timeout for playground tests, as it is pretty much one main test running - 'TestEndpointsReturnOk'. IOW, the blame-hang timeout would have to be at least as long as the full TestSession timeout.`

Is that clearer? I'm in two minds now whether to instead use `20min` blame-hang-timeout, which is the test session timeout, so it won't really serve any purpose. But if we add more tests and the session timeout goes up, then this hang timeout would become relevant.

Reply:
Adding in a follow up.